src/test/resources/articles-de/Verkehrspaedagoge.txt

```diff
@@ -0,0 +1,186 @@
+:[2] „Schon in Kitas und Schulen soll durch ''Verkehrspädagogen'' ‚eine Saat‘ für verantwortungsbewußtes Verhalten im Verkehr gelegt, die Problemgruppe der jungen Erwachsenen im Alter von 18 bis 25 Jahren, die die meisten Unfälle verursachen, sollen in ihrer eigenen Sprache angesprochen werden.“&lt;ref>{{Per-Tagesspiegel Online | Online=https://www.tagesspiegel.de/themen/brandenburg/polizei-setzt-auf-abschreckung-todesvideo-soll-rasern-eine-lehre-sein/62412.html | Autor=Alexander Pajevic | Titel= Polizei setzt auf Abschreckung: „Todesvideo“ soll Rasern eine Lehre sein| Tag=14 | Monat= 10| Jahr=1998 | Zugriff=2018-03-19}}&lt;/ref>
+:[2] „So sehr den ''Verkehrspädagogen'' Erfolg zu wünschen wäre – die vielfältigen Anreize zum Übertreten der Verkehrsregeln sind offenbar stärker.“&lt;ref>{{Per-Zeit Online | Online=http://www.zeit.de/1987/47/wenn-der-nebel-wieder-steigt/komplettansicht | Autor= Hans Schuh| Titel=  Wenn der Nebel wieder steigt …| Tag= 13| Monat=11| Jahr=1987 | Zugriff=2018-03-19}}&lt;/ref>
+:[2] „Angehende ''Verkehrspädagogen'' sind schon auf der Schulbank umworben.“&lt;ref> [http://fahrschul.tv/tag/verkehrspaedagogen/ Verkehrspädagogen]&lt;/ref>
+:[2] „Deshalb besteht seit Herbst 2014 für die erfolgreichen Absolventen der vorgesehenen Ausbildungskriterien die Möglichkeit, mit den jeweiligen Verbandsprüfungen abzuschließen und die Berechtigung zur Bezeichnung "Geprüfter ''Verkehrspädagoge'' / MPU-Berater (DVPK) bzw. "Geprüfter Kraftfahreignungsberater (DVPK)" (Deutscher Verband der Psychologen und Kraftfahreignungsberater e.V.)“&lt;ref>[http://www.mpu-beratung-vorbereitung.de/mpu_berater_ausbildung_kraftfahreignungsberater.html Ausbildung zum MPU-Berater, Fahreignungsberater, ''Verkehrspädagogen'']&lt;/ref>
+:[2] „Alexander K. ist seit 1992 Fahrlehrer und ''Verkehrspädagoge.'' Sein Fachwissen gibt er mit einem großen Maß an Kompetenz an seine Kursteilnehmer weiter.“&lt;ref>{{Internetquelle|url=https://www.fahrschule-keinath.de/ueber-uns-in-augsburg/das-keinath-team/alexander-keinath-teamleitung/|titel=Fahrschule, Augsburg|zugriff=2018-03-19}}&lt;/ref>
+
+{{Wortbildungen}}
+:[[Verkehrspädagogenausbildung]], [[Verkehrspädagogenkongress]]
+
+==== {{Übersetzungen}} ====
+{{Ü-Tabelle|Ü-links=
+*{{en}}: [1] {{Ü|en|}}
+*{{fr}}: [1] {{Ü|fr|}}
+*{{el}}: [1] {{Üt|el|}} 
+*{{it}}: [1] {{Ü|it|}}
+|Ü-rechts=
+*{{ru}}: [1] {{Üt|}}
+*{{sv}}: [1] {{Ü|}}
+*{{es}}: [1] {{Ü|es|}}
+*{{cs}}: [1] {{Ü|cs|}}
+}}
+
+{{Referenzen}}
+:[1] {{Wikipedia|Verkehrspädagoge}}
+:[*] {{Ref-DWDS|Verkehrspädagoge}}
+
+{{Quellen}}
+</textarea><div class="templatesUsed"></div><p id="mw-returnto">Zurück zur Seite <a href="/wiki/Verkehrsp%C3%A4dagoge" title="Verkehrspädagoge">Verkehrspädagoge</a>.</p>
```

Comment:
Please remove html code that does not belong to the wiki markup (i.e., everything from `</textarea>`

Reply:
Right, sorry. My bad.